src/app/services/progressesApi.ts

```diff
@@ -13,7 +14,12 @@ export const progressesApi = api.injectEndpoints({
                 },
             }),
         }),
+        userProgressDetails: builder.query<userDetails, string | undefined>({
+            query: (id): string => `/progresses?userId=${id}`,
+            providesTags: ['User_Standup'],
```

Comment:
we need to get all the standups of the user so that we can use each standup's timestamp to pass into getTotalMissedUpdates function to calculate the difference between the two dates.